docs/_panels/gitter.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Gitter
+description: metrics focused on Gitter messages.
+author: Nitish Gupta <imnitish.ng@gmail.com>
+screenshot: sigils/gitter.png
+created_at: 2020-04-18 (YYYY-MM-DD)
+grimoirelab_version: 0.2.0
+layout: panel
+---
+
+The Gitter panel shows information about messsages sent in a gitter room,
+over the course of it's age, gitter stores the text message sent alongwith
+the links in the form of username mentions, github or gitlab issues and
+pull requests which are automatically parsed as `@usename` and `#84` respectively
+by gitter. 
+
+Each user in a room is uniquely identified by a username and a user-id. This
+information is used to identify the most active and mentioned users.
+Similarly the links shared are identified as github issues or pull requests
+or categorized into URL hostnames if they belong to external sources.
```

Comment:
external or other sources?

Reply:
Other sources, other websites apart from gitlab or github. (stackoverflow, twitter etc)

Reply:
ok, thanks for the clarification. I would write it in this way:
```
.. if they belong to other sources (e.g., StackOverflow, Twitter).
```